### PR TITLE
Introduce BOM

### PR DESCRIPTION
**Pull request by sappenin**
A Bill of Materials (BOM) allows developers to control XRPL-4j versions from a single place.

Signed-off-by: David Fuelling <sappenin@gmail.com>

### CODE REVIEW

File: xrpl4j-bom/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                <version>1.0.1</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+</project>
+```
+
+With this in place, whenever you want add a `<dependency/>` you won't need to worr about specifying the version. Instead, version numbers are controlled by the BOM you import, as in the example above, which will use only version `1.0.1` of all XRPL-4j dependencies.
```
Comment (nkramer44) [inline xrpl4j-bom/README.md:31]:
```suggestion
With this in place, whenever you want add a `<dependency/>` you won't need to worry about specifying the version. Instead, version numbers are controlled by the BOM you import, as in the example above, which will use only version `1.0.1` of all XRPL-4j dependencies.
```

File: xrpl4j-bom/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+For more information on how BOM files work, consult this [tutorial](https://www.baeldung.com/spring-maven-bom) or other on Google.
```
Comment (nkramer44) [inline xrpl4j-bom/README.md:33]:
```suggestion
For more information on how BOM files work, consult this [tutorial](https://www.baeldung.com/spring-maven-bom) or others on Google.
```

**Review by nkramer44 — APPROVED** [review]
Just a couple typos, but this looks good

File: xrpl4j-bom/README.md (hunk 2 above)
Comment (sappenin) [reply xrpl4j-bom/README.md]: This is fixed.

**Pull request merged**